packages/amplication-git-utils/src/git/git.service.ts

```diff
@@ -89,39 +99,213 @@ export class GitClientService {
     }
 
     if (pullRequestMode === EnumPullRequestMode.Accumulative) {
-      await this.provider.createBranchIfNotExists({
+      const gitClient = new GitClient();
```

Comment:
wrap it with try/catch

Reply:
Why? i want it to crash if there is no git client